spec/data-model/README.md

```diff
@@ -46,6 +46,14 @@ the JSON and DTD definitions are intended for interchange between systems and pr
 To that end, they relax some aspects of the data model, such as allowing
 declarations, options, and attributes to be optional rather than required properties.
 
+> [!NOTE]
+> Users relying on XML representations of messages should note that
+> XML 1.0 does not allow for the representation of all C0 control characters (U+0000-U+0019),
+> and XML 1.1 does not allow for the representation of a U+0000 NULL character.
+> These characters are allowed in MessageFormat 2 messages,
+> so systems and users relying on XML as a resource format might need to
+> apply additional restrictions on message characters.
```

Comment:
... or provide an alternative escape mechanism for these characters.

RDF* for example uses `\u` and `\U` notations to allow non-XML chars. Note that by design our syntax does not provide these domain-specific escapes... exactly so that domains *can* provide them.

Reply:
```suggestion
> so systems and users relying on XML as a resource format might need to
> apply additional restrictions on message characters
> or provide an alternative escape mechanism for these characters.
```